api/src/main/java/com/github/eyefloaters/console/api/service/TopicService.java

```diff
@@ -307,7 +314,7 @@ List<OffsetSpec> getRequestOffsetSpecs(String offsetSpec) {
         return specs;
     }
 
-    List<org.apache.kafka.common.TopicPartition> getTopicPartitions(Map<Uuid, Either<Topic, Throwable>> topics, Map<String, Uuid> topicIds) {
+    Map<TopicPartition, List<Integer>> topicPartitionReplicas(Map<Uuid, Either<Topic, Throwable>> topics, Map<String, Uuid> topicIds) {
```

Comment:
It seems weird to me that this takes both some `topics` and also some `topicIds`. So the contract of this method is not clear from the signature alone and a sentence of Javadoc would be useful.

Reply:
Added a comment to `topicPartitionReplicas`. The `topicIds` arg should only be temporary (hopefully) until topic ID is supported by more operations.